api.rb

```diff
@@ -100,6 +100,11 @@ class Api < Roda
       render_with_games
     end
 
+    r.on 'title', String do |title|
+      request.params['title'] = title
```

Comment:
why do you do this?

Reply:
To force the initial set of games shown on the title page to be that title. Otherwise the initial set is all games.